Docs/articles/Installation/Running.md

```diff
@@ -22,26 +23,18 @@ the typical Cosmos developer provides more than enough functionality.
 
 VMWare Player can be downloaded [ here](http://vmware.com/download/player/).
 
-###  VMWare Server
-
-&nbsp;_place_holder;
-
 ###  QEMU
 
 Not officially supported at this time, but can be done. Just use the ISO option above with debugging turned off.
 
-Bochs
-
-Support is underway.
+### Bochs
 
-PXE
+Bochs is officially supported.
 
-For PXE, no special software is required. Only an ethernet connection to your router. However, if you have a computer that does not support network booting, or if you must perform the action over WiFi, you may need to look at a PXE chainloader such as [gPXE](#).
+### PXE
 
-Virtual PC
-
-Not officially supported at this time, but can be done. Just use the ISO option above with debugging turned off.
+For PXE, no special software is required. Only an ethernet connection to your router. However, if you have a computer that does not support network booting, or if you must perform the action over WiFi, you may need to look at a PXE chainloader such as [gPXE](http://etherboot.org/wiki/).
```

Comment:
It should explain more on how to use PXE, currently it only says that via ethernet but its not showing how to do that.